lib/main.dart

```diff
@@ -9,11 +10,14 @@ class MainApp extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return const MaterialApp(
+    return  MaterialApp(
+      title: 'weather app',
```

Comment:
recommend changing the debug banner
```suggestion
      title: 'weather app',
      debugShowCheckedModeBanner: false,
```